config/config.go

```diff
@@ -66,14 +66,16 @@ anybar-port: {{.AnybarPort}}
 keen-write-key: {{.KeenWriteKey}}
 keen-project-id: {{.KeenProjectId}}
 keen-event-collection: {{.KeenEventCollection}}
-environment: {{.Namespace}}`)
+environment: {{.Namespace}}
+# Do not change the kubernetes-config-key. If it has been changed please run the setup command again
+kubernetes-config-key: {{.Namespace}}`)
 	err = tmpl.Execute(f, config)
 	return err == nil
 }
 
 //takes the application config from viper and checks that all the required fields are populated
 func Validate() (n int, missing []string) {
-	var mandatorySettings = [7]string{"project-key", "remote-branch", "remote-name", "cluster-ip", "username", "password", "kubernetes-config-key"}
+	var mandatorySettings = []string{"project-key", "remote-branch", "remote-name", "cluster-ip", "container", "username", "password", "kubernetes-config-key"}
```

Comment:
The container isn't currently mandatory in the bash version. At the moment a container setting is needed for some of the commands so if it is not in the config it needs to be provided as a flag. It would be easier to make it mandatory and I think it would normal to have it so maybe it should be made mandatory and the related question in the setup command changed to reflect this.